packages/components/src/spinner/styles.ts

```diff
@@ -26,6 +26,8 @@ export const StyledSpinner = styled.svg`
 	position: relative;
 	color: ${ COLORS.ui.theme };
 	overflow: visible;
+	opacity: 1;
+	background-color: none;
```

Comment:
`none` is not a valid value for `background-color`. We can use `transparent` instead